server/server.go

```diff
@@ -549,23 +557,33 @@ func (s *server) AddNode(ctx context.Context, req *rpcpb.AddNodeRequest) (*rpcpb
 	if req.StartRequest.LogLevel != nil {
 		logLevel = *req.StartRequest.LogLevel
 	}
-	if req.StartRequest.WhitelistedSubnets != nil {
-		whitelistedSubnets = *req.StartRequest.WhitelistedSubnets
-	}
-	if req.StartRequest.PluginDir != nil {
-		pluginDir = *req.StartRequest.PluginDir
-	}
+
+	// use same configs from other nodes
+	whitelistedSubnets = s.network.nodeInfos[s.network.nodeNames[0]].WhitelistedSubnets
```

Comment:
probably shall add whitelistedSubnets and pluginDir fields to localNetwork struct

Reply:
Decided to store the whole options, because those were the actual initialization params.